packages/mobile_app/lib/components/prophecy_record.dart

```diff
@@ -11,51 +13,99 @@ class ProphecyRecord extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final changesCount = prophecy.changes.length;
```

Comment:
It was confusing again -- I was trying to recall what *changes* are and it took me 5 min to remember.

Reply:
OK, will rename to impactsCount, but I thought  `ImpactModel change in prophecy.changes` will be understandable enough.

Reply:
Done.